roadmap.yaml

```diff
@@ -60,7 +62,9 @@ Data Engineering:
     - Replication
     - ACID
     - Normalization or Denormalization
-    - Batch and Stream processing
+    - Batch and Stream processing:
+        - KSQL
+        - Kafka Stream
```

Comment:
Let's review the roadmap again, which is accessible through [this link](https://raw.githubusercontent.com/data-burst/data-engineering-roadmap/main/statics/roadmap.png). Here we don't introduce any tools. The tools are introduced in its dedicated section (data processing).

Reply:
I removed them.